options.js

```diff
@@ -0,0 +1,34 @@
+document.addEventListener('DOMContentLoaded', function() {
+	console.log('closing tab set0.');
+	selectCurrentValues();
+    let saveButton = document.getElementById('save');
```

Comment:
to chyba nie musi być w eventListenerze, i w zasadzie to chyba niepotrzebnie rozbite na dwa